src/render_cmd.rs

```diff
@@ -287,6 +312,30 @@ impl RenderCmd {
         }
         Ok(buffer)
     }
+
+    ///  Filter out the deployment values that match the regex
+    fn filter_deployment(&self, regex: String, name: &String, values: Option<Vec<PathBuf>>) -> Option<Vec<PathBuf>> {
+        let is_name_filtered = self.is_name_filtered(regex, &name);
+
+        if is_name_filtered {
+            return values;
+        }
+        return Some(vec![]);
+    }
+
+    
+
+    /// Check if deployment name is filtered
+    fn is_name_filtered(&self, regex: String, name: &String) -> bool {
+        let regex_string = format!("[{}]{}", name, "{4,}");
+        let is_match = Regex::new(&regex_string.to_string()).unwrap().is_match(&regex.to_string());
+
+        if is_match {
```

Comment:
In Rust, the last statement of a function is the return value, hcen you can save on the creation of `is_match` and just let the last line of your function be `Regex::new(&regex_string.to_string()).unwrap().is_match(&regex.to_string())`. Mind how it does not end with a semicolon, thereby signaling to `rustc` that it is to be used as return value.